frontend/src/components/grantaccess/form/ForestClientInput.vue

```diff
@@ -27,14 +27,22 @@ const emit = defineEmits([
 const forestClientNumbersInput = ref('');
 const forestClientData = ref<FamForestClient[]>([]);
 const forestClientNumberVerifyErrors = ref([] as Array<string>);
+const verifiedNumbers: Set<string> = new Set();
```

Comment:
Hi Jota, we already has a method `isForestClientNumberNotAdded` in this component, so I think you just need to move the circled part in the screenshot to the place where before calling the api
<img width="765" alt="image" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/21b62175-f8cd-401b-b4aa-e1ffbf574ee8">

Reply:
hello! I'll make the change